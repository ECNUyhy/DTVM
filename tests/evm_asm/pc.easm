// return memory[4:36]: 0000000000000000000000000000000000000000000000000000000000000002
PC          // PC=0x00(PUSH1 0x00)
PUSH1 0x02
ADD
PC          // PC=0x04(PUSH1 0x04)
MSTORE      // store to memory: offset=0x04, value=0x02
PUSH1 0x20
PUSH1 0x04
RETURN      // return: offset=0x04, size=0x20
