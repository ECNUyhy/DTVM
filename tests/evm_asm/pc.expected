status: success
error_code: 0
stack: []
memory: '00000000000000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000'
storage: {}
transient_storage: {}
return: '0000000000000000000000000000000000000000000000000000000000000002'
events: []
